index.html

```diff
@@ -5,25 +5,197 @@
+          It is our great pleasure to welcome you to the "5th International
+          conference on Astronomy, Astrophysics and Space Science" which is
+          going to be held during June 27-28, 2022 at Addis Ababa, Ethiopia.
+        </p>
+        <h3>June 27-28, 2022 Addis Ababa, Ethiopia</h3>
+        <p class="location">
+          &#64; 4-kilo main hall, Addis Ababa University.
+        </p>
+      </div>
+    </section>
+
+    <!-- Program-section -->
+    <section class="program" id="programs">
+      <h2 class="program-heading">Main Program</h2>
+      <div class="underline"></div>
+      <div class="program-items">
+        <div class="item">
+          <img
+            src="./images/icons-1.svg"
+            alt="icons-1"
+            class="item-icon"
+          />
+          <h3 class="program-title">Humanoid robots</h3>
+          <p class="program-description">
+            Like all service robots, they provide 
+            value by automating tasks in a way that leads 
+            to cost-savings and productivity
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-2.svg"
+            alt="icon-2"
+            class="item-icon"
+          />
+          <h3 class="program-title">Human-Robot Interaction</h3>
+          <p class="program-description">
+            Robots have evolved from continuous human-controlled 
+            master–slave servomechanisms for handling nuclear waste.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-3.svg"
+            alt="icon-3"
+            class="item-icon"
+          />
+          <h3 class="program-title">Applications</h3>
+          <p class="program-description">
+            RI is now applied in almost all robot tasks, including manufacturing
+            , space, aviation, undersea, surgery, rehabilitation, agriculture, education.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-4.svg"
+            alt="icon-4"
+            class="item-icon"
+          />
+          <h3 class="program-title">Human supervisory control</h3>
+          <p class="program-description">
+            robots in performance of routine tasks. These include 
+            handling of parts on manufacturing assembly lines.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="./images/icons-5.svg"
+            alt="icon-5"
+            class="item-icon"
+          />
+          <h3 class="program-title">Human–robot social interaction</h3>
+          <p class="program-description">
+            robot devices to provide entertainment, teaching, comfort, 
+            and assistance for children and elderly, autistic, and handicapped persons.
+          </p>
+        </div>
+      </div>
+      <a href="#" class="conference-held"> <span>Join BDU International conference 2022</span></a>
+    </section>
```

Comment:
- [ ] As from the Behance template, kindly make sure when a card has hovered, it should have an interactive styling (e.g.: a border, increased size,...)
### Figma Template image
![Screenshot from 2022-07-28 03-37-41](https://user-images.githubusercontent.com/83042742/181401618-da28b91f-1804-45b6-8d7a-49a2f4f2cc19.png)